sdk/core/System.ClientModel/src/Pipeline/ClientRetryPolicy.cs

```diff
@@ -0,0 +1,238 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.ClientModel.Internal;
+using System.Collections.Generic;
+using System.IO;
+using System.Runtime.ExceptionServices;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace System.ClientModel.Primitives;
+
+public class ClientRetryPolicy : PipelinePolicy
+{
+    private const int DefaultMaxRetries = 3;
+    private static readonly TimeSpan DefaultInitialDelay = TimeSpan.FromSeconds(0.8);
+
+    private readonly int _maxRetries;
+    private readonly TimeSpan _initialDelay;
+
+    public ClientRetryPolicy() : this(DefaultMaxRetries)
```

Comment:
should we hide this ctor and expose shared Default policy through a static property?

Reply:
What about applying a default value to the `maxRetries` parameter in the other constructor and just removing this?

Reply:
Doing both

Reply:
Addressed